functional_tests.go

```diff
@@ -6551,6 +6425,8 @@ func main() {
 
 		// SSE-C tests will only work over TLS connection.
 		if tls {
+			testEncryptionPutGet()
+			testEncryptionFPut()
```

Comment:
can we add a test for FGetEncryptedObject?

Reply:
Will add test 👍